### DISCUSSION TITLE
Add utility / transfer fees to be taken into account when comparing hourly prices.

### DISCUSSION THREAD

**Discussion by Ruokoton** [kotope/aio_energy_management#51] [Ideas]
Not sure if this is Finland specific thing, but transfer / utility fees can be significant portion of the total hourly price, so it would make sense to offer option to take these expenses into account in the hourly price comparisons.

I think there are three different schemes for that :

General transfer (flat rate)
Seasonal transfer (1 Nov-31 Mar, Mon-Sat 23:00 - 7:00 high tariff, low tariff during other times)
Night transfer (Mon-Sun 07:00-22:00 high tariff, low tariff during other times)

Would make sense to have all those switching points configurable, as well as the tariffs themselves.

I would welcome input from other users, because I am toying with idea of contributing to the project with this feature, unless someone beats me to it (or I am convinced it's a bad idea).

**Comment by chacques** [comment]
I wonder if this is already taken account in nordpool integration? I think it had some sort of possibility to add extra costs.

**Reply by chacques** [reply]
More info here: https://github.com/custom-components/nordpool?tab=readme-ov-file#additional-costs

**Comment by Ruokoton** [comment]
Yeah, seems this is needless. Almost disappointed. 😅

**Comment by kotope** [comment]
Getting back to this 'old' thread..

As MTU 15min is getting closer and official Nord Pool integration will be the way to go with it. There is also no support in the official integration with the additional costs unfortunately.

Therefore The next AIO Energy Management release 0.5.0 will have price_modifications configuration with custom template support.  🎉
